modules/ibmz-configure-devices-mco.adoc

```diff
@@ -0,0 +1,263 @@
+   name: 99-worker0-devices
+spec:
+   config:
+     ignition:
+       version: 3.2.0
+     storage:
+       files:
+       - contents:
+           source: data:text/plain;base64,<encoded_base64_string> <2>
+         filesystem: root
+         mode: 420
+         path: /etc/udev/rules.d/41-zfcp-host-0.0.8000.rules <3>
+----
+<1> The role you have defined in the machine config file.
+<2> The Base64 encoded string that you have generated in the previous step.
+<3> The path where the udev rule is located.
+
+[id="configuring-fcp-lun"]
+== Configuring an FCP LUN
+The following is an example of how to configure an FCP LUN by adding a udev rule. You can add new FCP LUNs or add additional paths to LUNs that are already configured with multipathing.
+
+.Procedure
+
+. Take the following sample udev rule `41-zfcp-lun-0.0.8000:0x500507680d760026:0x00bc000000000000.rules`:
++
+[source,terminal]
+----
+ACTION=="add", SUBSYSTEMS=="ccw", KERNELS=="0.0.8000", GOTO="start_zfcp_lun_0.0.8207"
+GOTO="end_zfcp_lun_0.0.8000"
+
+LABEL="start_zfcp_lun_0.0.8000"
+SUBSYSTEM=="fc_remote_ports", ATTR{port_name}=="0x500507680d760026", GOTO="cfg_fc_0.0.8000_0x500507680d760026"
+GOTO="end_zfcp_lun_0.0.8000"
+
+LABEL="cfg_fc_0.0.8000_0x500507680d760026"
+ATTR{[ccw/0.0.8000]0x500507680d760026/unit_add}="0x00bc000000000000"
+GOTO="end_zfcp_lun_0.0.8000"
+
+LABEL="end_zfcp_lun_0.0.8000"
+----
+
+. Convert the rule to Base64 encoded, by running the following command:
++
+[source,terminal]
+----
+$ base64 /path/to/file/
+----
+
+. Copy the following MCO sample profile into a YAML file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1 
+kind: MachineConfig 
+metadata:
+   labels:
+     machineconfiguration.openshift.io/role: worker0 <1>
+   name: 99-worker0-devices
+spec:
+   config:
+     ignition:
+       version: 3.2.0
+     storage:
+       files:
+       - contents:
+           source: data:text/plain;base64,<encoded_base64_string> <2>
+         filesystem: root
+         mode: 420
+         path: /etc/udev/rules.d/41-zfcp-lun-0.0.8000:0x500507680d760026:0x00bc000000000000.rules <3>
+----
+<1> The role you have defined in the machine config file.
+<2> The Base64 encoded string that you have generated in the previous step.
+<3> The path where the udev rule is located.
+
+[id="configuring-dasd"]
+== Configuring DASD
+
+The following is an example of how to configure a DASD device by adding a udev rule.
+
+.Procedure
+
+. Take the following sample udev rule `41-dasd-eckd-0.0.4444.rules`:
++
+[source,terminal]
+----
+ACTION=="add", SUBSYSTEM=="ccw", KERNEL=="0.0.4444", DRIVER=="dasd-eckd", GOTO="cfg_dasd_eckd_0.0.4444"
+ACTION=="add", SUBSYSTEM=="drivers", KERNEL=="dasd-eckd", TEST=="[ccw/0.0.4444]", GOTO="cfg_dasd_eckd_0.0.4444"
+GOTO="end_dasd_eckd_0.0.4444"
+
+LABEL="cfg_dasd_eckd_0.0.4444"
+ATTR{[ccw/0.0.4444]online}="1"
+
+LABEL="end_dasd_eckd_0.0.4444"
+----
+
+. Convert the rule to Base64 encoded, by running the following command:
```

Comment:
Comma could be removed here, eg: 

. Convert the rule to Base64 encoded by running the following command: